### PR TITLE
Allow lazy update option.

### PR DESCRIPTION
**Pull request by YooSunYoung**

### CODE REVIEW

File: src/plopp/widgets/drawing.py [hunk 1]
```diff
@@ -55,6 +63,7 @@ def __init__(
         destination: FigureLike | Node,
         get_artist_info: Callable,
         value: bool = False,
+        lazy_update: bool = False,
```
Comment (YooSunYoung) [inline src/plopp/widgets/drawing.py:66]: Can we have this option in the constructor?
(I opened a PR to share the idea.)

Comment (nvaytet) [reply src/plopp/widgets/drawing.py:66]: Good idea. Should we use the `ipywidgets` terminology (which some users may already know)?; i.e. use `continuous_update = True` instead?

Comment (YooSunYoung) [reply src/plopp/widgets/drawing.py:66]: Sure. I wasn't aware of the common terminology thanks!

Comment (YooSunYoung) [reply src/plopp/widgets/drawing.py]: @nvaytet Done...!

I just tested in the notebook. Seems working as expected.

https://github.com/user-attachments/assets/b813bcd9-eba4-4138-b113-2ef1fca5490f

**Review by nvaytet — APPROVED** [review]

**Pull request merged**